Video.js

```diff
@@ -279,7 +279,7 @@ export default class Video extends Component {
     const isAsset = !!(uri && uri.match(/^(assets-library|ipod-library|file|content|ms-appx|ms-appdata):/));
 
     let nativeResizeMode;
-    const RCTVideoInstance = this.getViewManagerConfig('RCTVideo');
+    const RCTVideoInstance = this.getViewManagerConfig('RNVVideo');
```

Comment:
If we're changing RCTVideo to RNVVideo, we should probably rename the const here too

Reply:
Done!